openhands/datasets/isolated/bhosporus22k.py

```diff
@@ -0,0 +1,46 @@
+import os
+import pandas as pd
+from glob import glob
+from .base import BaseIsolatedDataset
+from ..data_readers import load_frames_from_video
+
+class Bhosporus22kDataset(BaseIsolatedDataset):
+    """
+    Turkis Isolated Sign language dataset(Bhosporus22k) from the paper:
+    Link to paper: https://arxiv.org/pdf/2004.01283.pdf
+    """
+    def read_glosses(self):
+        df = pd.read_csv(self.class_mappings_file_path, delimiter=",")
+        print(self.class_mappings_file_path)
+        all_glosses = [df.iloc[i][2] for i in range(len(df))]
+        unique_glosses=set(all_glosses)
+        self.glosses = list(unique_glosses)
+        #print(self.glosses)
+
+    def read_original_dataset(self):
+        """
+        Dataset includes 3200 videos where 10 non-expert subjects executed 5 repetitions of 64 different types of signs.
+
+        For train-set, we use signers 1-8.
+        Val-set & Test-set: Signer-9 & Signer-10
```

Comment:
1. Can you please add a comment here about what you actually found out about the `signer_id`s?
2. What signer_id did the SOTA paper use? (which you said had 93% accuracy)
3. Can you please write an email to the Bhosporus asking if the signer_id you found out (for both val & test) is the correct thing?

Reply:
4. Why does the description say val-set & test-set if there's no val-set? Please edit it if this is incorrect.